.github/workflows/itk_dict.txt

```diff
@@ -715,6 +715,7 @@ orthonormality
 opencl
 openjpeg
 oper
+osterior
```

Comment:
This is word does not belong to the dictionary, because it should be rejected. It would be better to avoid using this word in the source code or disable spellchecking at the single place where this word is intentionally used.